presidio-analyzer/presidio_analyzer/__main__.py

```diff
@@ -68,7 +69,15 @@ def serve_command_handler(enable_trace_pii,
     logger.info("GRPC started")
 
     logger.info("Creating RecognizerRegistry")
-    registry = RecognizerRegistry()
+    enable_text_analytics_recognizer = \
+        os.environ.get('ENABLE_TEXT_ANALYTICS_RECOGNIZER')
+    if enable_text_analytics_recognizer is not None \
+            and (enable_text_analytics_recognizer != ''
+                 or enable_text_analytics_recognizer == 'True'
+                 or enable_text_analytics_recognizer == '1'):
+        registry = RecognizerRegistry(RecognizerStoreApi(), None, True)
```

Comment:
It would be more readable if you just passed the enable_text_analytics_recognizer value to the RecognizerRegistry constructor. No need for if/else here. 
```suggestion
        registry = RecognizerRegistry(recognizer_store_api=RecognizerStoreApi(),
                                                        recognizers=None,
                                                        enable_text_analytics_recognizer=enable_text_analytics_recognizer)
```

Reply:
`enable_text_analytics_recognizer` is an environment variable, and is string value. this is why we need to parse the value. 
I am not sure `RecognizerRegistry` should accept the flag as string but rather as boolean.
Wdyt?

Reply:
Should be cast to boolean. My note was mostly around how to call the RecognizerRegistry. The current call is hard to understand when reading through the code